### PR TITLE
Search image meeting

### PR DESCRIPTION
**Pull request by DanitzaMachicado**
Add accept button on modal view and change logic

### CODE REVIEW

**Comment by rodrigouz** [comment]
Fix conflicts

File: src/main/webapp/WEB-INF/foodmeeting/searchImage.jsp [hunk 1]
```diff
@@ -5,11 +5,11 @@
 	<div class="image-links">
 		<c:forEach var="image" items="#{images}">
 			<div class="grid-item"
-				style="width:200px">
+				 style="width:200px">
```
Comment (rayaviri) [inline src/main/webapp/WEB-INF/foodmeeting/searchImage.jsp]: white space

File: src/main/webapp/WEB-INF/foodmeeting/searchImage.jsp [hunk 2]
```diff
@@ -5,11 +5,11 @@
...
 				<div class="card">
 					<div class="card-image waves-effect waves-block waves-light">
 						<img class="image-link" data-image-link="${image}"
-							src="${image}">
+							 src="${image}">
```
Comment (rayaviri) [inline src/main/webapp/WEB-INF/foodmeeting/searchImage.jsp]: white space

File: src/main/webapp/WEB-INF/foodmeeting/searchImage.jsp [hunk 3]
```diff
@@ -22,17 +22,20 @@
 			<div class="card">
 				<div class="card-image">
 					<img id="image-card-id"
-						src="http://mainefoodstrategy.org/wp-content/uploads/2015/04/HealthyFood_Icon.jpg"
-						class="materialboxed">
+						 src="http://mainefoodstrategy.org/wp-content/uploads/2015/04/HealthyFood_Icon.jpg"
```
Comment (rayaviri) [inline src/main/webapp/WEB-INF/foodmeeting/searchImage.jsp]: white space

File: src/main/webapp/WEB-INF/foodmeeting/searchImage.jsp [hunk 4]
```diff
@@ -22,17 +22,20 @@
...
+						 class="materialboxed">
 				</div>
 			</div>
 		</div>
 		<div class="col m12 l8">
 			<div class="input-field col s12">
 				<input name="image-link" id="image-link-id"
-					value="http://mainefoodstrategy.org/wp-content/uploads/2015/04/HealthyFood_Icon.jpg"
-					type="text" class="validate" autofocus="autofocus"> <label for="image-link">ImageLink</label>
+					   value="http://mainefoodstrategy.org/wp-content/uploads/2015/04/HealthyFood_Icon.jpg"
```
Comment (rayaviri) [inline src/main/webapp/WEB-INF/foodmeeting/searchImage.jsp]: white space

File: src/main/webapp/WEB-INF/foodmeeting/settingMeeting.jsp [hunk 5]
```diff
@@ -13,11 +13,18 @@
             $(function() {
                 $('.datepicker').pickadate({
                     selectMonths: true,
-                    selectYears: 15
+                    selectYears: 15,
+					closeOnSelect: true,
```
Comment (rayaviri) [inline src/main/webapp/WEB-INF/foodmeeting/settingMeeting.jsp:17]: indentation

File: src/main/webapp/assets/js/foodMeetings.js [hunk 6]
```diff
@@ -8,6 +8,10 @@ $(function () {
 
 	var foodMeetingsList = new FoodMeetingsList(foodMeetingsContainer, $newMeetingPlaceholder);
 	foodMeetingsList.init();
+	
+	var modal = new ModalSearchImage($("#image-card-id"), $("#image-link-id"),
+					$('.image-link'), $('.image-links'), $("#image-card-id"));
+				modal.init();
```
Comment (rayaviri) [inline src/main/webapp/assets/js/foodMeetings.js]: indentation

File: src/main/webapp/assets/js/foodMeetings.js [hunk 7]
```diff
@@ -35,10 +39,9 @@ $(function () {
 	});
 
 	communicationService.connect('ws://' + location.host + '/ws', createMeetingRoomId);
-
 	var newFoodMeeting = new NewFoodMeeting(foodMeetingsContainer, createMeetingRoomId, communicationService, $newMeetingPlaceholder, $newMeeting);
-
 	newFoodMeeting.init();
+
```
Comment (rayaviri) [inline src/main/webapp/assets/js/foodMeetings.js]: remove this line

**Review by rayaviri — APPROVED** [review]
looks fine

**Pull request merged**